.github/workflows/ci_run_scm_rts.yml

```diff
@@ -150,6 +150,7 @@ jobs:
         cd ${SCM_ROOT}
         ./contrib/get_all_static_data.sh
         ./contrib/get_thompson_tables.sh
+        ./contrib/get_aerosol_climo.sh
```

Comment:
This is going to slow down CI a lot. Any way to stash this data for future use?

Reply:
I was worried that this was going to bring this whole thing crashing down, but to my surprise it is faster to download this data on the GitHub servers than on Hera.[ It only took ~2min.](https://github.com/NCAR/ccpp-scm/actions/runs/9195025937/job/25289863152?pr=475), but that's more than the ~15sec before.
There may be slick ways to stash this, I just don't know any.